src/XacroLoader.d.ts

```diff
@@ -11,7 +11,7 @@ interface XacroLoaderOptions {
 
 export class XacroLoader {
 
-    load(url: string, onLoad: (xml: XMLDocument) => void, options?: XacroLoaderOptions): Promise<XMLDocument>;
-    parse(content: string, onLoad: (xml: XMLDocument) => void, options?: XacroLoaderOptions): Promise<XMLDocument>;
+    load(url: string, onLoad: (xml: XMLDocument) => void, onError: (err: Error) => void): void;
+    parse(content: string, onLoad: (xml: XMLDocument) => void, options?: XacroLoaderOptions): void;
```

Comment:
Thanks for the fix! If we're here can we update the `parse` function with the correct signature, as well? It should be the same change as for `load`. See here:

https://github.com/gkjohnson/xacro-parser/blob/master/src/XacroLoader.js#L41